modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/UserNotificationSettings.tsx

```diff
@@ -58,33 +64,21 @@ export function UserNotificationSettings({
 	const [toTerms, setToTerms] = useState<string>('');
 	const [userList, setUserList] = useState<MultiSelectItem[]>([]);
 
-	const getUserNotificationRoles = async () => {
+	const queryRoles = async () => {
```

Comment:
Hey @rijema, We normally don't use the `query` word for functions that will get something from the API, so you could rename this function to `getRoles` and move this function to `rolesUtils.ts`?

To do this you can move the const `HEADERS` to `util/constants.tsx` and export it from there, you can also take the opportunity to rename this file extension to `constants.ts` since we are not returning any JXS from there.

By moving this function to `roleUtils.ts` you can also move the `Roles` and `Role` interfaces there.

Reply:
Just did it.